core/v0/api/core.json

```diff
@@ -2894,33 +2895,7 @@
                         "$ref": "#/components/schemas/Billing"
                     },
                     "fulfillment": {
-                        "allOf": [
-                            {
-                                "$ref": "#/components/schemas/Fulfillment"
-                            },
-                            {
-                                "properties": {
-                                    "customer": {
-                                        "allOf": [
-                                            {
-                                                "$ref": "#/components/schemas/Person"
-                                            },
-                                            {
-                                                "$ref": "#/components/schemas/Contact"
-                                            },
-                                            {
-                                                "required": [
-                                                    "name"
-                                                ]
-                                            }
-                                        ]
-                                    }
-                                },
-                                "required": [
-                                    "customer"
-                                ]
-                            }
-                        ]
+                        "$ref": "#/components/schemas/Fulfillment"
```

Comment:
I suggest you merge the `customer` schema with the `Fulfillment` schema. That will remove `allOf` and keep the spec compatible with existing implementations.

Reply:
Please, check: https://github.com/beckn/protocol-specifications/pull/66/commits/d732bda0dc08e4d76b86517183dcce34a644f39f

Reply:
But I am pretty sure we discussed previously that we want to remove info about customer from here.

Reply:
Discussed on the call. Decided move customer object inside Fulfillment schema.

Reply:
Please, check: https://github.com/beckn/protocol-specifications/pull/66/commits/323e127e51cf6bcfbb49596247765a676bb46339